src/main/java/software/bernie/techarium/pipes/capability/IPipeNetworkManagerCapability.java

```diff
@@ -1,7 +1,25 @@
 package software.bernie.techarium.pipes.capability;
 
 import net.minecraft.nbt.CompoundNBT;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.server.ServerWorld;
 import net.minecraftforge.common.util.INBTSerializable;
+import org.jetbrains.annotations.NotNull;
+import software.bernie.techarium.pipes.networks.PipeNetwork;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
 
 public interface IPipeNetworkManagerCapability extends INBTSerializable<CompoundNBT> {
+
+    void tick(ServerWorld world);
+
+    @NotNull
+    List<PipeNetwork> getNetworks();
+
+    void addNetwork(PipeNetwork network);
+
+    <T extends PipeNetwork> Optional<T> getNetwork(PipeType type, BlockPos pos);
+    <T extends PipeNetwork> Optional<T> getByUUIDorPos(UUID uuid, BlockPos pos);
```

Comment:
Could we split this up into two methods and add javadocs? I got no clue what this does.

Reply:
these do nothing at the moment. They were just an idea I had to find the network at a position or using an UUID which could be part of the network, so at serialization of the pipetileentity it could just store the uuid and changes to the network would be reflected, even if the tileentity wasn't loaded at the time those changes were made. Evaluation of usefullness of these methods will be done when the pipe blocks/tileentites are added

Reply:
Lookup by uuid would be faster too, because this way not all networks need to be checked to find the network at a given position, but instead could be found using a Map